### PR TITLE
change the toast component content props from string to react-node

### PR DESCRIPTION
**Pull request by Seolhun**
## Self Checklist

- [x] I wrote a PR title in **English**.
- [x] I added an appropriate **label** to the PR.
- [x] I wrote a commit message in **English**.
- [x] I wrote a commit message according to [**the Conventional Commits specification**](https://www.conventionalcommits.org/en/v1.0.0/).
- [x] [I added the appropriate **changeset**](https://github.com/channel-io/bezier-react/blob/next-v1/CONTRIBUTING.md#add-a-changeset) for the changes.
- [x] I wrote **a unit test** about the implementation.
- [x] I wrote **a storybook document** about the implementation.
- [ ] I tested the implementation in **various browsers**.
  - [ ] Windows: Chrome, Edge, (Optional) Firefox
  - [x] macOS: Chrome, Edge, Safari, (Optional) Firefox

# Summary

- change the toast component content props from string to react-node

# Details

- Replace the line breaking "\n" to div element in string content.
- Changed from string to ReactNode to build a custom content toast.
- Since ReactNode is string compatible, there is no breaking-changes.
- Fixed a problem broader than the Close area than the Content area. (flex-shrink)
- actionContent props is deprecated. It is used one place only in the desk application.
- Add a CustomContent(ReactNode) storybook.

Screen Shot 2022-07-25 at 14 10 08

Screen Shot 2022-07-24 at 22 07 31

https://user-images.githubusercontent.com/16330024/180923853-adb35cf9-c7eb-4801-9ccc-496b45d6de9c.mov

## Browser Compatibility
OS / Engine 호환성을 반드시 확인해주세요.
### Windows
- [ ] Chrome - Blink
- [ ] Edge - Blink
- [ ] Firefox - Gecko (Option)
### macOS
- [x] Chrome - Blink
- [x] Edge - Blink
- [x] Safari - WebKit
- [x] Firefox - Gecko (Option)

# References

https://github.com/channel-io/ch-desk-web/issues/10597
- [Figma](https://www.figma.com/file/82ho83cCj4n1FbD9ErU825/Contacts?node-id=4210%3A150677)
- [Notion - Contacts Bulk Action](https://www.notion.so/channelio/DB-BulkAction-1668569383ae412abf2b3e2cd479f73d)

### CODE REVIEW

**Comment by Seolhun** [comment]
토스트 Custom Content 텍스트 바꿔야겠네요.

**Review by sungik-choi — APPROVED** [review]
- [changeset을 추가](https://github.com/channel-io/bezier-react/blob/next-v1/CONTRIBUTING.md#add-a-changeset)해주세요(영문)! semantic-release에서 changeset으로 변경하면서, 커밋으로 릴리즈 노트를 자동으로 생성해주지 않기 때문에 버전업에 관련된 변경사항에 대해 개발자가 직접 추가해주어야합니다.
- PR 제목을 영문으로 변경해주시면 감사하겠습니다 🙏 오픈소스임을 고려해서 PR 제목 등은 영문으로 작성하는 방식으로 맞추려고 해요(#895)

**Review by CHEWCHEWW — APPROVED** [review]

File: .changeset/witty-bananas-return.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@channel.io/bezier-react": patch
+---
+
+refactor(toast): change toast content props "string" to "react-noe"
```
Comment (sungik-choi) [inline .changeset/witty-bananas-return.md:5]:
```suggestion
refactor(toast): change toast content props "string" to "react-node"
```

typo

**Review by sungik-choi — APPROVED** [review]

File: packages/bezier-react/src/components/Toast/ToastProvider.tsx [hunk 2]
```diff
@@ -123,7 +124,7 @@ function ToastProvider({
           autoDismissTimeout={autoDismissTimeout}
           preset={preset}
           appearance={appearance}
-          content={content}
+          content={content || ''}
```
Comment (quino0627) [inline packages/bezier-react/src/components/Toast/ToastProvider.tsx:127]: ToastElement에서 default로 ''를 채워주는데, fallback이 필요한가요?
https://github.com/channel-io/bezier-react/pull/894/commits/f8481ac1405c835a87c2355a75e23b100ec93b8d#diff-85ac2201c8cd87922f19fcf93d3b985de850225c6ea38d267ca63ab0f4a6a99aR21

Comment (Seolhun) [reply packages/bezier-react/src/components/Toast/ToastProvider.tsx:127]: 어제는 잘 안보였는데, 다시보니까 Required(ToastElementProps에서)랑 ReactNode에서 차이가 발생된거였네요.
```tsx
export type ToastContent = NonNullable<React.ReactNode>
```
로 변경하면 될것 같습니다.

**Review by quino0627 — APPROVED** [review]

**Pull request merged**